packages/blade/src/components/Link/_decisions/decisions.md

```diff
@@ -0,0 +1,64 @@
+# Link Decisions <!-- omit in toc -->
+- [Design](#design)
+- [Internal BaseLink Component](#internal-baselink-component)
+  - [BaseLink API](#baselink-api)
+- [Link Component](#link-component)
+  - [Link API](#link-api)
+- [Component Breakdown](#component-breakdown)
+- [Accessibility](#accessibility)
+- [Open Questions](#open-questions)
+
+## Design
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=12699%3A147155) to the **Link** component
+
+
+## Internal BaseLink Component
+Internal component that exposes certain extra props to enable creation of Component Level Action Link like `positive`, `negative`, `neutral`, `information`, `notice`
+
+### BaseLink API
+| Prop | Type | Required | Default | Description |
+|---|---|---|---|---|
+| variant | `anchor`, `button` | *No* | `anchor` | The variant of the Link to be rendered.<br><br>Note: `button` will look like an inline Link component (no spacing; can be rendered alongside other text) but will be rendered as a `button` element. |
+| intent |  `positive`,`negative`, `notice`, `information`, `neutral` | *No* | undefined | The intent of the link to be rendered.<br><br>  Note: <br> - We will use this to set the intent for internal Blade components that have positive, negative, notice, info, or neutral link buttons like an Alert component. |
+| contrast | `low`, `high` | *No* | `low` | The contrast of the link to be rendered.<br><br>  Note: We need this for internal Blade components since a `positive` intent could have a `high` as well as a `low` contrast link. |
+| children | `string` | When `icon` is passed: *No*<br>When `icon` is not passed: *Yes*   | undefined | The text to be rendered within the link. |
+| icon | `Icon` | When `children` is passed: *No*<br>When `children` is not passed: *Yes* | undefined | The Blade `Icon` component to be rendered within the link. |
+| iconPosition | `left`, `right` | *No* | `right` | The position of the rendered icon. |
+| isDisabled | `boolean` | *No* | `false` | Control whether the link is disabled or not.<br><br>Note: This prop is only valid for `button` variant. |
+| onClick | `Function` | No | undefined | The function to be called when the link is clicked. |
+| href | `string` | No | undefined | The href for the link component.<br><br>Note: This prop is only valid for `anchor` variant. |
+| target | `string` | No | undefined | The target for the link component.<br><br>Note: This prop is only valid for `anchor` variant. |
+| rel | `string` | No | undefined | The rel for the link component.<br><br>Note: This prop is only valid for `anchor` variant. |
```

Comment:
we can basically say if `target="_blank"` at that time we can define the default value as `noopener noreferrer`

Reply:
Shouldn't we keep it `undefined` by default? Similar to `a` tag?

Reply:
depends what we are solving for. here we solving for some better practices given to the consumers as out of the box while they use blade components vs a wrapper over `a` tag with just styles and then consumers decide what props they want to pass

Reply:
If we're modifying the HTML defaults then I feel it would make more sense to have a different prop like `isExternal` for setting both target and rel accordingly. Makes the intent clearer imo.

Reply:
but then how we will support other target values `target="_blank|_self|_parent|_top|framename"` if we add a boolean prop?

Reply:
Non blocker: yes, I think we can provide `target` and `rel` for overriding things. `isExternal` should be able to cover majority usecases.